### PR TITLE
add FAQ entry for compatable registries

### LINKED ISSUE #104: Registry Support: listed the compatible registries
- Azure Container Registry
- Docker Hub
- ORAS Distribution Registry 
- Zot registry

### PR DESCRIPTION
**Pull request by zr-msft**
Added FAQ entry with the list of compatible registries

Fixes #104

### CODE REVIEW

File: content/en/docs/faq.md
Comment (FeynmanZhou) [inline content/en/docs/faq.md:8]:
```suggestion
## What registries are compatible with Notary?
```

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]
PTAL

File: content/en/docs/faq.md
Comment (zr-msft) [reply content/en/docs/faq.md]: Apologies for the typo. Fixed

**Comment by zr-msft** [comment]
@FeynmanZhou incorporated your feedback

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Review by toddysm — APPROVED** [review]

**Review by toddysm — APPROVED** [review]

File: content/en/docs/faq.md
Comment (toddysm) [inline content/en/docs/faq.md:13]: Docker Hub is not compatible at the moment.

**Review by toddysm — CHANGES_REQUESTED** [review]

File: content/en/docs/faq.md
Comment (zr-msft) [reply content/en/docs/faq.md:13]: @toddysm Just to confirm, Docker has adopted `notation` per this: https://github.com/notaryproject/notaryproject.dev/pull/124/files#diff-a70d13bfec1091119e5df723c7213d64484d0a62069fe3c3f892a6418be523a2R19

but it is unavailable in dockerhub?

Comment (toddysm) [reply content/en/docs/faq.md:13]: Pushing Notation signatures to Docker Hub is failing as of right now. It used to work back in January.

Comment (FeynmanZhou) [reply content/en/docs/faq.md:13]: @toddysm Yes. I think we could remove DH at this moment.

**Review by yizha1 — DISMISSED** [review]
LGTM, we can add more later once we verified it, for example, Harbor

**Review by FeynmanZhou — DISMISSED** [review]
LGTM

File: content/en/docs/faq.md
Comment (toddysm) [inline content/en/docs/faq.md:13]: Can we add the exact location for download? Finding this on the web site may be cumbersome...

Comment (FeynmanZhou) [reply content/en/docs/faq.md:13]: @toddysm  Goot suggestion. @zr-msft Could you replace it with this URL: https://github.com/oras-project/distribution/pkgs/container/registry/64589674?tag=v1.0.0-rc.4

**Review by FeynmanZhou — COMMENTED** [review]
Added two compatible registries in the comment

**Comment by zr-msft** [comment]
@FeynmanZhou @toddysm updated links. 

@FeynmanZhou I'm holding off on including Harbor until they release 2.8.0

Please take a look when you have a chance.

File: content/en/docs/faq.md
Comment (zr-msft) [reply content/en/docs/faq.md:17]: included GH, but holding off on Harbor until they release 2.8.0

**Review by FeynmanZhou — DISMISSED** [review]
LGTM

**Comment by FeynmanZhou** [comment]
@toddysm @vaninrao10 @priteshbandi @iamsamirzon could you help review it? Thanks

**Review by toddysm — APPROVED** [review]

**Review by iamsamirzon — DISMISSED** [review]
Suggest 
1. changing the title to "What registries are compatible with Notation"
2. adding Amazon ECR as it supports OCI 1.0 based approach for storing signatures.

**Comment by FeynmanZhou** [comment]
> Suggest
> 
> 1. changing the title to "What registries are compatible with Notation"
> 2. adding Amazon ECR as it supports OCI 1.0 based approach for storing signatures.

@iamsamirzon Could you confirm that you have tested Notation v1.0.0-RC.3 with ECR and it works fine?

**Comment by iamsamirzon** [comment]
> > Suggest
> > 
> > 1. changing the title to "What registries are compatible with Notation"
> > 2. adding Amazon ECR as it supports OCI 1.0 based approach for storing signatures.
> 
> @iamsamirzon Could you confirm that you have tested Notation v1.0.0-RC.3 with ECR and it works fine?

Yes. I have tested the basic workflows of signing, signing multiple times, signing with meta-data, verification, and notation inspect and ls commands.

**Comment by yizha1** [comment]
@zr-msft could you update the doc based on @iamsamirzon comment. Thanks.

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by priteshbandi — APPROVED** [review]

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**
